exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Murmur128.java

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
We already have a Murmur Hash implementation for use with Direct buffers. (see `org.apache.drill.exec.expr.fn.impl.MurmurHash3` ). Can we merge these two and have only one copy?